rudder-server/SOURCES/Makefile

```diff
@@ -161,7 +161,8 @@ install: build rudder-server-version initial-promises initial-ncf
 	mkdir -p $(DESTDIR)/var/rudder/run
 	mkdir -p $(DESTDIR)/var/rudder/ldap/openldap-data
 	mkdir -p $(DESTDIR)/var/log/rudder/apache2/
-	mkdir -p $(DESTDIR)/var/log/rudder/webapp
+	mkdir -p $(DESTDIR)/var/log/rudder/webapp/jetty
+	mkdir -p $(DESTDIR)/var/log/rudder/webapp/jvm
```

Comment:
perhpas also `mkdir -p $(DESTDIR)/var/log/rudder/webapp/api` depending of what we decide for that directory

Reply:
Since we added webapp/api, the additional mkdir is needed